internal/cli/atlas/commonerrors/errors_test.go

```diff
@@ -36,13 +34,14 @@ func TestCheck(t *testing.T) {
 			err:  nil,
 			want: nil,
 		},
-		{
+		// TODO: Update the err object to test errors.go file
+		/* {
```

Comment:
If we have follow up jira lets add it here.

Reply:
Then that error instance will print exactly the same output

err.Error() == errClusterUnsupported

Reply:
fields of admin.GenericOpenApiError are unexported field hence this does not work :(

Reply:
I will address this unit test as part of [CLOUDP-185092](https://jira.mongodb.org/browse/CLOUDP-185092). Thank You!